### PR TITLE
DP-703 have frontend and debs with multi prod ability

### PR DESCRIPTION
**Pull request by duartecoelhomovai**
Tests done:

https://github.com/MOV-AI/movai_drivers_joysticks/actions/runs/2208319707

Screenshot from 2022-04-22 15-36-00
Screenshot from 2022-04-22 15-36-15

### CODE REVIEW

File: .github/workflows/py-deb-workflow.yml [hunk 1]
```diff
@@ -154,7 +158,7 @@ jobs:
     needs: [Build]
     runs-on: ubuntu-20.04
     container:
-      image: registry.aws.cloud.mov.ai/devops/ros-buildtools-noetic:latest
+      image: registry.aws.cloud.mov.ai/qa/ros-buildtools-noetic:latest
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/py-deb-workflow.yml:161]: `devops`, `qa`, `develop`, `prod` we have too many namespaces I think, one should be enough

File: .github/workflows/ros-workflow.yml [hunk 2]
```diff
@@ -272,6 +245,7 @@ jobs:
     - name: Commit raise version
       id: raise
       run: |
+        git config --global --add safe.directory $(pwd)
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/ros-workflow.yml:248]: what is that for ?

**Review by AlexFernandes-MOVAI — APPROVED** [review]
nothing blocking, this is really nice and will ease up the migration to the new Nexus, now I need to migrate all repos before merging to branch `v1`

File: .github/workflows/py-deb-workflow.yml (hunk 1 above)
Comment (duartecoelhomovai) [reply .github/workflows/py-deb-workflow.yml:162]: yep true.

File: .github/workflows/ros-workflow.yml (hunk 2 above)
Comment (duartecoelhomovai) [reply .github/workflows/ros-workflow.yml:248]: Git just released a new security rule. 
https://github.blog/2022-04-12-git-security-vulnerability-announced/


Screenshot from 2022-04-22 15-32-39

**Pull request merged**